build-support/bin/cherry_pick.sh

```diff
@@ -44,22 +55,25 @@ if [[ -z $CATEGORY_LABEL ]]; then
   read -r CATEGORY_LABEL
 fi
 REVIEWERS=$(gh pr view "$PR_NUM" --json reviews --jq '.reviews.[].author.login' | sort | uniq)
-BODY_FILE=$(mktemp "/tmp/github.cherrypick.$PR_NUM.$MILESTONE.XXXXXX")
-PR_CREATE_CMD=(gh pr create --base "$MILESTONE" --title "$TITLE (Cherry-pick of #$PR_NUM)" --label "$CATEGORY_LABEL" --body-file "$BODY_FILE")
-while IFS= read -r REVIEWER; do PR_CREATE_CMD+=(--reviewer "$REVIEWER"); done <<< "$REVIEWERS"
-BRANCH_NAME="cherry-pick-$PR_NUM-to-$MILESTONE"
 
-if [[ -z $COMMIT ]]; then
-  fail "Wasn't able to retrieve merge commit for $PR_NUM."
-fi
+for MILESTONE in $MILESTONES; do
+  BODY_FILE=$(mktemp "/tmp/github.cherrypick.$PR_NUM.$MILESTONE.XXXXXX")
+  PR_CREATE_CMD=(gh pr create --base "$MILESTONE" --title "$TITLE (Cherry-pick of #$PR_NUM)" --label "$CATEGORY_LABEL" --body-file "$BODY_FILE")
+  while IFS= read -r REVIEWER; do PR_CREATE_CMD+=(--reviewer "$REVIEWER"); done <<< "$REVIEWERS"
+  BRANCH_NAME="cherry-pick-$PR_NUM-to-$MILESTONE"
+
+  if [[ -z $COMMIT ]]; then
+    fail "Wasn't able to retrieve merge commit for $PR_NUM."
+  fi
 
-gh pr view "$PR_NUM" --json body --jq '.body' > "$BODY_FILE"
-git fetch https://github.com/pantsbuild/pants "$MILESTONE"
-git checkout -b "$BRANCH_NAME" FETCH_HEAD
-git cherry-pick "$COMMIT" ||
-  fail "\nPlease fix the above conflicts, commit, and then run:\n  ${PR_CREATE_CMD[*]}"
+  gh pr view "$PR_NUM" --json body --jq '.body' > "$BODY_FILE"
+  git fetch https://github.com/pantsbuild/pants "$MILESTONE"
```

Comment:
There's currently a 2.17.x milestone, but no `2.17.x` branch. Will this attempt to cherry pick to that and fail to run the `git fetch`?